File: C/helloWorld.c
#include<stdio.h>

/* fn must be declared before calling main() :O */
const char* printHelloWorld() {

	/* Return "Hello World"*/
	return "Hello World";

}

void main() {

	printf("%s\n", printHelloWorld());


	for (int i = 0; i <= 10; i++) {
		if (i != 10) {
			printf("%d, ", i);
		} else {
			printf("%d\n", i);
		}
	}
}

File: golang/helloWorld.go
package main

import "fmt"

func main() {
	fmt.Println("Hello World")
	arrayPrint()
}

func arrayPrint() {
	var arr [10]int

	for i := 0; i <= len(arr)-1; i++ {
		arr[i] = i + 1

		if i != len(arr)-1 {
			fmt.Printf("%d, ", arr[i])
		} else {
			fmt.Printf("%d\n", arr[i])
		}
	}

}
